src/harness/test_harness_objects.py

```diff
@@ -206,6 +208,103 @@ def registerCbsdsAndRequestGrants(self,
                            [grant_response['grantId']], [grant_request])
         self.cbsd_objects[cbsd_id] = cbsd_object
 
+  def registerCbsdsAndRequestGrantsWithPpa(self,
+                                           registration_requests,
+                                           grant_requests,
+                                           ppa_record,
+                                           cluster_list,
+                                           conditional_registration_data=None):
+    """Construct CBSD object based on the result of registration and
+    grant requests.
```

Comment:
Maybe updating comment to specify that PPA Record is also Injected?

Reply:
Done, thanks!